common/src/main/java/com/microsoft/hydralab/performance/entity/IOSEnergyGaugeInfo.java

```diff
@@ -61,4 +63,14 @@ public class IOSEnergyGaugeInfo implements Serializable {
     private float locationOverhead;
     @JSONField(name = "energy.thermalstate.overhead")
     private float thermalStateOverhead;
+
+    @Override
+    public LinkedHashMap<String, Double> getBaselineMetricsKeyValue() {
+        return null;
+    }
+
+    @Override
+    public SummaryType getSummaryType() {
+        return null;
+    }
```

Comment:
@dinghong2233  please override these methods for iOS battery and memory entity at your convenience.